airflow_munchkin/class_parser.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+import inspect
+import textwrap
+from typing import Dict, List, NamedTuple, Optional
+
+from airflow_munchkin.docstring_parser import GoogleDocstringParser, DocField
+
+
+class ClassParameterInfo(NamedTuple):
+    name: str
+    kind: str
+    desc: List[str]
```

Comment:
By kind we understand operator / hook / sensor? And `desc` stands for descendants?

Reply:
Description.

Reply:
Example value for kind: `int`, `str`, `Union[A, B]`, etc.

Reply:
Got it!